Library/Formula/watch.rb

```diff
@@ -12,15 +18,47 @@ class Watch < Formula
     sha1 "a7c559378bc74cd30d00f962e63d6ee5c705aea1" => :mountain_lion
   end
 
+  depends_on "gettext" => :build
```

Comment:
That would become something like:

```ruby
if build.with? "nls"
  depends_on "gettext"
else
  depends_on "gettext" => :build
end
```
But that's clunky and to be honest I'd rather we avoid it by either not making the NLS option available until we resolve the build issues, or resolving the build issues & enabling it by default with no off.